kotlin/openID4VP/src/commonMain/kotlin/io/mosip/openID4VP/jwt/jwe/JWEHandler.kt

```
@@ -18,28 +21,25 @@ class JWEHandler(
+            )
+                .keyID(publicKey.kid)
+                .agreementPartyUInfo(Base64URL(walletNonce))
+                .agreementPartyVInfo(Base64URL(verifierNonce))
+                .build()
+
+            val encrypter = EncryptionProvider.getEncrypter(publicKey)
+            val jweObject = JWEObject(header, Payload(payloadString))
+            jweObject.encrypt(encrypter)
+
+            return jweObject.serialize()
         } catch (exception: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)